### ISSUE TITLE
No slideshows in Greece

### ISSUE THREAD

**Issue by andimik** [Opendigitalradio/dablin#36]
Even with `-L` I don't get any slideshow from ERT Greece. There are lots of [X-PAD len] [X-PAD len] messages without `-L` ...

[ERT_20180506.eti.zip](https://github.com/Opendigitalradio/dablin/files/1977768/ERT_20180506.eti.zip)

Xpadxpert, qt-dab and welle.io show them indeed.

![proto-icon-144x144](https://user-images.githubusercontent.com/24510556/39675309-85df6f82-5159-11e8-8074-1737aeaaeb8d.png)
![deftero-icon-96x96](https://user-images.githubusercontent.com/24510556/39675310-8606122c-5159-11e8-8ae2-d166d0bcf08c.png)
![trito-icon-96x96](https://user-images.githubusercontent.com/24510556/39675311-8631c8b8-5159-11e8-8784-0a6bc46c1163.png)
![eraspor-icon-96x96](https://user-images.githubusercontent.com/24510556/39675312-8659de02-5159-11e8-8722-2fc733b3fedf.png)
![kosmos-icon-96x96](https://user-images.githubusercontent.com/24510556/39675313-868122a0-5159-11e8-8209-e06070f6061f.png)
![voiceofgr-icon-96x96](https://user-images.githubusercontent.com/24510556/39675314-86a349fc-5159-11e8-9f49-d4e91110bf4b.png)

**Comment by basicmaster** [comment]
I looked into the dump and it turned out that unfortunately the Slideshow signalling is missing for the affected channels. As XPADxpert works without that signalling (as it is in the FIC, which is not necessarily contained in the file types it can open), the slides are shown there. But this is definitely a misconfiguration on broadcaster side.

**Comment by basicmaster** [comment]
I will try to contact ERT about that and get back to you.

**Comment by andimik** [comment]
Ok, good luck ;-)

**Comment by andimik** [comment]
Also the Turkish DAB slides do not appear in dablin_gtk ...

See 
[trt.eti.zip](https://github.com/Opendigitalradio/dablin/files/2449442/trt.eti.zip)

Please process this ETI file using

`cat /tmp/trt.eti | odr-dabmod ini_file.ini > /tmp/trt.raw`
and then 
`eti-cmdline-rawfiles -F /tmp/trt.raw | dablin_gtk`

The direct way playing the initial eti does not work. 

> ETIPlayer: ignored ETI frame with ERR = 0x55


My ini_file.ini is

```
[input]
transport=file
source=/dev/stdin

[modulator]
gainmode =1
rate=2048000

[output]
output=file

[fileoutput]
format=u8
filename=/dev/stdout
```

**Comment by basicmaster** [comment]
In the TRT case it is again the missing slideshow signalling. I will contact them next week.

Regarding ERT, if I remember correctly, you said that they don't have a slideshow anymore. Is this correct?

**Comment by andimik** [comment]
No, in the meanwhile they re-appeared, but let me check later again.

**Comment by jpuigs** [comment]
Turkish TRT radio channels have slideshows in all channels except for "World".

**Comment by andimik** [comment]
Do they now work in dablin_gtk?

 That's what I reported. I know that they have SLS, but there seems to be no signalling, so it might be ignored by tools.

**Comment by basicmaster** [comment]
Exactly: All channels except "+World" have a Slideshow, but the mandatory signalling is missing. I contacted a TRT colleague earlier and hope to get an answer/change soon.

**Comment by basicmaster** [comment]
@jpuigs
According to TRT, the signalling should be fixed now. Hence DABlin should show the Slideshow. Could you please check?

@andimik
Regarding the original issue with the ERT mux...is the issue still present?

**Comment by jpuigs** [comment]
Yes, they appear !!!
![imagen](https://user-images.githubusercontent.com/44906458/50849602-cfcf0e00-1377-11e9-9f9c-2d59ff312081.png)


Sometimes, when zapping, it appear these audio problems seen on console.
Closing and opening again dablin fixes it.

**Comment by basicmaster** [comment]
> Yes, they appear !!!

Great!

> Sometimes, when zapping, it appear these audio problems seen on console.
> Closing and opening again dablin fixes it.

So it starts and also continues when you switch the service? Does this also apply to other muxes? Does it also happen when you first wrote the ETI stream to a file, and later play it with DABlin?

> ERT:

Hmm, could you provide a 1-2min ETI dump of this mux?

**Comment by andimik** [comment]
Could you also upload a current Turkish ETI-file? The one you posted some days seems not to be fixed by TRT and therefore I cannot see the slides in dablin.

**Comment by jpuigs** [comment]
The audio problems I had mainly on English Muxes (4.8 East) seem to be fixed.
The Satellite transpoder where these 3 muxes are, has a FEC of 7/8 , and I had a C/N of about 8 dB.
Some days ago I changed the dish which I received the 4.8 East with, and now I have 10.5 dB C/N and I haven't seen audio problems anymore.

**Comment by andimik** [comment]
Well, I have sometimes SDLOutput errors, but no ignored ETI frames.

I have 14dB SNR with an 80 cm dish in Central Europe.

**Comment by jpuigs** [comment]
I use a 80 cm. dish too, but you are "near" the 60 cm. zone, and I'm inside the 90 cm. one....

**Comment by andimik** [comment]
This is the output of 24 hrs listening to Heart 80 (I'm very impressed, good music choice) as attachment. No ignored ETI frames at all.

[sdl_24hrs.txt](https://github.com/Opendigitalradio/dablin/files/2766028/sdl_24hrs.txt)

And, you are right. It's strange: UK radio DAB feeds use a footprint which is better for Eastern Europe than for UK ...

**Comment by basicmaster** [comment]
Regarding the audio buffer overflows, I opened #45.

Regarding the "ignored ETI frame" errors, I only have the explanation of bad reception, as already mentioned...

**Comment by andimik** [comment]
Not only bad reception. If you stream from a STB the bandwidth has to be wide enough...

Unfortunately the Symbol Rate on 7 East has increased, so that may lead to the same error.

**Comment by jpuigs** [comment]
No, I have no problems at 7 East.

When streaming from STB, it just gets streamed the PID which carries the DAB ETI signal, not the whole transponder.

**Comment by basicmaster** [comment]
All these INFO/DEBUG/WARN messages indicate that the stream is already received disrupted by the respective software, before it is forwarded to DABlin. Hence the DABlin fix (from #45) only addresses the implications i.e. the delayed stream. So I can do nothing more about it.

Regarding ERT, I today got the response that the signalling is planned to be fix in a few month as part of larger changes. So regarding DABlin, the issue is fixed so far and I therefore close here.

**Comment by andimik** [comment]
Cool, thanks. Hope they will use better pics like http://webradio.ert.gr/wp-content/uploads/2015/06/trito.png
